### PR TITLE
Check if the state variable with the specific key exists in app's schema

### PR DESCRIPTION
**Pull request by StylishTriangles**

### CODE REVIEW

File: pytealext/state.py [hunk 1]
```diff
@@ -66,6 +72,10 @@ def put(self, value: Expr) -> App:
     def get(self) -> App:
         return App.localGet(Int(0), self._name)
 
+    def exists(self) -> Expr:
+        mb = App.localGetEx(Int(0), Int(0), self._name)
+        return Seq(mb, mb.hasValue())
+
```
Comment (Sernikjamnika) [inline pytealext/state.py:78]: do you think it's worth adding test for that? e.g. 
```
If(mb.exists()).Then(Approve()).Else(Reject())
```

Additionally what does mb stands for?

Comment (StylishTriangles) [reply pytealext/state.py:78]: mb for maybe of the maybevalue

File: pytealext/evaluator/evaluator.py [hunk 2]
```diff
@@ -486,13 +499,25 @@ def eval_teal(
             b = stack.pop()
             a = stack.pop()
             if a != 0:
-                raise Panic(
+                raise Exception(
```
Comment (StylishTriangles) [inline pytealext/evaluator/evaluator.py:502]: Exception describes our evaluator's limitation, Panic resembles a runtime panic in TEAL

File: pytealext/state.py (hunk 1 above)
Comment (StylishTriangles) [reply pytealext/state.py]: changed mb to ex

File: pytealext/evaluator/evaluator.py [hunk 3]
```diff
@@ -474,6 +474,19 @@ def eval_teal(
...
+            if context is None:
+                raise Exception("app_global_get_ex requires execution environment context")
+            if app != 0:
+                raise Exception("Accessing other app's global state is unsupported")
+            if not isinstance(key, bytes):
+                raise Panic("app_global_get_ex key must be a bytes value", current_line)
+            val = context.global_state.get(key, 0)
+            exists = key in context.global_state
+            stack.append(val)
+            stack.append(exists)
```
Comment (Sernikjamnika) [inline pytealext/evaluator/evaluator.py:489]: bool to int :)

File: pytealext/evaluator/evaluator.py [hunk 4]
```diff
@@ -486,13 +499,25 @@ def eval_teal(
...
+            app = stack.pop()
+            account = stack.pop()
+            if app != 0 or account != 0:
+                raise Exception("app_local_get_ex is only supported with 0" " as the account and application parameter")
+            if not isinstance(key, bytes):
+                raise Panic("app_local_get_ex key must be a bytes value", current_line)
+            val = context.local_state.get(key, 0)
+            exists = key in context.local_state
+            stack.append(val)
+            stack.append(exists)
```
Comment (Sernikjamnika) [inline pytealext/evaluator/evaluator.py:520]: ditto

**Review by Sernikjamnika — APPROVED** [review]

**Pull request merged**